### ISSUE TITLE
JASP results DWLS - Discrepancies with Lavaan

### ISSUE THREAD

**Issue by bruno-fd** [jasp-stats/jasp-issues#1259]
* JASP version: 0.14.1.0
*OS name and version: Windows 10

Hello all,

I´m trying to figure out why fit indexes using Robust DWLS in JASP are, in general so good! 
And why they are so discrepant in comparison to lavaan results using WLSMV.  

When we request WLSMV in lavaan, it comes out with two colums (DWLS and Robust). So, If im not wrong, lavaan collumn 'robust' should be the same of JASP Robust DWLS, or am I missing something?  Does anyone has any tip on this?

Thanks in advance.    

I believe that part of this result is due to the fact that JASP still does not allow categorical variables yet.

If that´s true, than I have another question: is is 'correct' to use DWLS in 'metric' variables?

Thanks,

Bruno

**Comment by platjavi** [comment]
Hello, i have a problem with the program on linux and Windows, when i try to do a  clasical one sample t-test I get a message that says the following: 
Known bug: computed effect sizes ignore the test value. Could you help me to solve the problem?


Javier

**Comment by boutinb** [comment]
> Hello, i have a problem with the program on linux and Windows, when i try to do a clasical one sample t-test I get a message that says the following:
> Known bug: computed effect sizes ignore the test value. Could you help me to solve the problem?
> 
> Javier

Hello @platjavi If you have an issue, please make a new one, and don't use an existing one: we use GitHub issues precisely to organize the issues. The warning you see is just to say that we know there is a bug in this analysis, and that will be fixed in the next release.

**Comment by EJWagenmakers** [comment]
The bug is discussed in various places on this GitHub page. @AlexanderLyNL could you point to a few relevant entries?

**Comment by AlexanderLyNL** [comment]
The following provides some background: https://github.com/jasp-stats/jasp-issues/issues/1173

**Comment by boutinb** [comment]
This issue is about a Lavaan problem from @bruno-fd. If @platjavi wants to add some comments, he should make another issue.
@Kucharssim could you answer the Lavaan problem?

**Comment by LSLindeloo** [comment]
@bruno-fd 

I assume the following is the reason for these discrepancies:

> For the DWLS and ULS estimators, lavaan also provides ‘robust’ variants: WLSM, WLSMVS, WLSMV, ULSM, ULSMVS, ULSMV. Note that for the robust WLS variants, we use the diagonal of the weight matrix for estimation, but we use the full weight matrix to correct the standard errors and to compute the test statistic.

I will add the robust estimators to all analyses that use lavaan, so you can expect these in the next release!

**Comment by LSLindeloo** [comment]
@bruno-fd 
I think I have figured out the issue; 

so... WLSMV is DWLS estimation with robust standard errors and a scaled and shifted test statistic. 

However, when the additional fit indices were requested, the non-scaled and shifted indices were displayed.

I have now changed this, such that the fit indices show the 'scaled and shifted' values when the selected model test is "scaled and shifted". 

I hope this provides some clarification.

If you have any questions, feel free to ask!

Lorenzo
